core/body/body.ctrl.js

```diff
@@ -132,12 +132,12 @@ export class BodyCtrl extends View {
 				this.navigate(cell);
 				if (cell.column.editorOptions.trigger === 'click' && this.view.edit.cell.enter.canExecute(cell)) {
 
-					if (edit().method === 'batch' && this.selection.items.length > 1) {
-						this.model.selection({items: []});
+					if (edit().startCell) {
+						edit({startCell: null});
```

Comment:
you should memorize startCell, in cell-handler.component, you can get it there defenetly :)